backend/Dockerfile

```diff
@@ -1,15 +1,19 @@
-FROM php:7.4.2-apache
+FROM php:7.4-apache
 
 WORKDIR /app
 
-RUN apt-get -y update && apt-get -y upgrade && apt-get -y install libxml2-dev curl unzip iproute2 libonig-dev
```

Comment:
Sehr cool dass die inzwischen alle nicht mehr extra installiert werden müssen! Das Dockerfile ansonsten hast du hübsch gemacht, was eigentlich nicht ganz den [Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers) entspricht (mehrere zusammengehörige `RUN` commands könnte man in einen einzelnen kombinieren). Aber mit deinen Änderungen läufts schon mal schneller als in meiner Version, darum okay für den Moment denke ich.

Reply:
Ah, war mir nicht bewusst dass jedes `RUN` einen neuen layer erstellt. Kann ich beim nächsten Anpassen allenfalls noch etwas optimieren. Die einzelnen Features (z.b. MySQL vs. XDebug) würde ich glaubs aber getrennt als separate Layers lassen.